<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormKaryawan</class>
 <widget class="QWidget" name="FormKaryawan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>281</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineNik"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineNama"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboJk"/>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineAlamat"/>
    </item>
    <item>
     <widget class="QComboBox" name="cAgama"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineNotlp"/>
    </item>
    <item>
     <widget class="QLineEdit" name="linePendidikan"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineKjabatan"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>160</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nik</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nama Karyawan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Jenis Kelamin</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Tanggal Lahir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Alamat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Agama</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>No Telpon</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Pendidikan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Kode Jabatan</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>522</width>
     <height>87</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnSimpan">
      <property name="text">
       <string>SIMPAN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnUbah">
      <property name="text">
       <string>EDIT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBersih">
      <property name="text">
       <string>BERSIH</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHapus">
      <property name="text">
       <string>HAPUS</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>321</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>KARYAWAN PERUSAHAAN</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableKaryawan">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>521</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>nik</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>nama_karyawan</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>jk</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>tgl_lahir</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>alamat_karyawan</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>agama</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>no_hp</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>pendidikan</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>kd_jabatan</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>158</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Cari Karyawan</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editcari">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>550</y>
     <width>279</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
